docs/sphinx_doc/en/source/tutorial/209-rag.md

```diff
@@ -0,0 +1,126 @@
+(209-rag-en)=
+
+# A Quick Introduction to RAG in AgentScope
+
+We want to introduce three concepts related to RAG in AgentScope: Knowledge, KnowledgeBank and RAG agent.
```

Comment:
If the document if for AgentScope developers, we should avoid to use the concepts in LlamaIndex (e.g. nodes, docs, etc), or provide pre knowledge here (if necessary).

Reply:
Tried to update the terms/words to avoid requiring too much knowledge from LlamaIndex